zb-xiaomi.js

```diff
@@ -476,6 +477,129 @@ const MODEL_IDS = {
       },
     },
   },
+  'lumi.plug.maeu01': {
+    name: 'smartplug',
+    '@type': [
+      'SmartPlug',
+      'EnergyMonitor',
+      'OnOffSwitch',
+      'MultiLevelSensor',
+    ],
+    activeEndpoints: {
+      1: {
+        profileId: PROFILE_ID.ZHA_HEX,
+        inputClusters: [
+          CLUSTER_ID.GENONOFF_HEX,
+          CLUSTER_ID.SEMETERING_HEX,
+          CLUSTER_ID.HAELECTRICAL_HEX,
+        ],
+        outputClusters: [],
+      },
+    },
+    properties: {
+      switch: {
+        descr: {
+          '@type': 'OnOffProperty',
+          label: 'On/Off',
+          type: 'boolean',
+        },
+        profileId: PROFILE_ID.ZHA,
+        endpoint: 1,
+        clusterId: CLUSTER_ID.GENONOFF,
+        attr: 'onOff',
+        setAttrFromValue: 'setOnOffValue',
+        parseValueFromAttr: 'parseOnOffAttr',
+      },
+      instantaneousPower: {
+        descr: {
+          '@type': 'InstantaneousPowerProperty',
+          label: 'Power',
+          type: 'number',
+          unit: 'watt',
+          readOnly: true,
+        },
+        profileId: PROFILE_ID.ZHA,
+        endpoint: 1,
+        clusterId: CLUSTER_ID.HAELECTRICAL,
+        attr: 'activePower',
+        parseValueFromAttr: 'parseNumericTenthsAttr',
+      },
+      counter: {
+        descr: {
+          '@type': 'InstantaneousPowerProperty',
```

Comment:
This property should not have the `InstantaneousPowerProperty` `@type`, as it's not the instantaneous power.

Reply:
Do you have a suggestion which one will be better suited?

Reply:
No, you can just omit that field.

Reply:
Alright.